acls.tf

```diff
@@ -0,0 +1,35 @@
+module "acl_label" {
+  source     = "git::https://github.com/cloudposse/tf_label.git?ref=tags/0.2.0"
+  namespace  = "${var.namespace}"
+  stage      = "${var.stage}"
+  name       = "${var.name}"
+  delimiter  = "${var.delimiter}"
+  attributes = "${var.attributes}"
+  tags       = "${var.tags}"
+}
+
+resource "aws_network_acl" "default" {
+  count      = "${var.create_network_acl ? 1 : 0}"
+  vpc_id     = "${var.vpc_id}"
+  subnet_ids = ["${compact(concat(aws_subnet.public.*.id, aws_subnet.private.*.id))}"]
+
+  egress {
+    rule_no    = 100
```

Comment:
Have you confirmed this is before the default rule which is to deny everything?

Reply:
Yes. The `Deny ALL` rule always is on the last line